Readability.js

```diff
@@ -1737,7 +1763,7 @@ Readability.prototype = {
         var contentLength = this._getInnerText(node).length;
 
         var haveToRemove =
-          (img > 1 && img > p && !this._hasAncestorTag(node, "figure")) ||
+          (img > 1 && p / img < 0.5 && !this._hasAncestorTag(node, "figure")) ||
```

Comment:
I think it will do for now, though I also notice that some of the `<img>` tags in the source don't have a `src` attribute. We should probably remove those images and not count them here, which would (I think?) also fix this with a bit less risk to regressing something else for which we might not have a testcase. But let's do that in a followup, this set of commits is confusing enough as it is.